app/src/main/java/ch/hikemate/app/model/authentication/FirebaseAuthRepository.kt

```diff
@@ -79,7 +79,7 @@ class FirebaseAuthRepository : AuthRepository {
             onSuccess(auth.currentUser)
           } else {
             Log.d("FirebaseAuthRepository", "signInWithCredential:failure")
-            onErrorAction(task.exception ?: Exception())
+            onErrorAction(R.string.error_occurred_while_signing_in_with_google)
```

Comment:
Request: Could you maybe add a logging of the exception?

Reply:
Yep

Reply:
Thanks!